src/RecoveryServices/RecoveryServices.Backup.Helpers/Conversions/RetentionPolicyConversions.cs

```diff
@@ -67,9 +67,6 @@ public partial class PolicyHelpers
                 ltrPolicy.YearlySchedule = GetPSLTRYearlySchedule(serviceClientRetPolicy.YearlySchedule, timeZone);
             }
 
-            // safe side validate
-            ltrPolicy.Validate();
```

Comment:
@siddharth7 what's the reason for removing this?

Reply:
The validation for policies is not supporting the existing policies and is throwing an error. We have decided to go away with validation on get-policy command